chart/templates/deployment.yaml

```diff
@@ -191,8 +191,16 @@ spec:
 {{- if eq .Values.auditLog.destination "sidecar" }}
   {{- if gt (int .Values.auditLog.level) 0 }}
       # Make audit logs available for Rancher log collector tools.
-      - image: {{ .Values.busyboxImage }}:{{.Values.busyboxImageTag}}
-        imagePullPolicy: {{ default "IfNotPresent" .Values.busyboxImagePullPolicy }}
+      {{- if .Values.busyboxImage }}
```

Comment:
I was thinking that everyone upgrading would have this set, but I suppose only if you had overridden the default. We should make sure to write upgrade as a test case.